### ISSUE TITLE
unexpected keyword argument 'pullup'

### ISSUE THREAD

**Issue by ilium007** [mcauser/micropython-mcp23017#7]
Getting this error when following the list interface example
```
>>> mcp[0].input(pullup=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unexpected keyword argument 'pullup'
>>> 
```

**Comment by ilium007** [comment]
Example should be changed to:

`>>> mcp[0].input(pull=1)`

**Comment by mcauser** [comment]
Hi @ilium007 Thanks for that.
Fixed in e7cea6e1a98b54a4fe5b0d8e4e8c2f549259caa1
